install-dev/data/db_structure.sql

```diff
@@ -1,2630 +1,2821 @@
-  `id_shop` INT UNSIGNED NULL DEFAULT NULL,
-  `id_shop_group` INT UNSIGNED NULL DEFAULT NULL,
-  `id_carrier` int(10) unsigned NOT NULL,
-  `id_range_price` int(10) unsigned DEFAULT NULL,
-  `id_range_weight` int(10) unsigned DEFAULT NULL,
-  `id_zone` int(10) unsigned NOT NULL,
-  `price` decimal(20,6) NOT NULL,
-  PRIMARY KEY (`id_delivery`),
-  KEY `id_zone` (`id_zone`),
-  KEY `id_carrier` (`id_carrier`,`id_zone`),
-  KEY `id_range_price` (`id_range_price`),
+  `id_delivery` int(10) unsigned NOT NULL auto_increment, 
+  `id_shop` INT UNSIGNED NULL DEFAULT NULL, 
+  `id_shop_group` INT UNSIGNED NULL DEFAULT NULL, 
+  `id_carrier` int(10) unsigned NOT NULL, 
+  `id_range_price` int(10) unsigned DEFAULT NULL, 
+  `id_range_weight` int(10) unsigned DEFAULT NULL, 
+  `id_zone` int(10) unsigned NOT NULL, 
+  `price` decimal(20, 6) NOT NULL, 
+  PRIMARY KEY (`id_delivery`), 
+  KEY `id_zone` (`id_zone`), 
+  KEY `id_carrier` (`id_carrier`, `id_zone`), 
+  KEY `id_range_price` (`id_range_price`), 
   KEY `id_range_weight` (`id_range_weight`)
 ) ENGINE=ENGINE_TYPE DEFAULT CHARSET=utf8 COLLATION;
 
 /* Admin users */
 CREATE TABLE `PREFIX_employee` (
-  `id_employee` int(10) unsigned NOT NULL auto_increment,
-  `id_profile` int(10) unsigned NOT NULL,
-  `id_lang` int(10) unsigned NOT NULL DEFAULT '0',
-  `lastname` varchar(32) NOT NULL,
-  `firstname` varchar(32) NOT NULL,
-  `email` varchar(128) NOT NULL,
-  `passwd` varchar(60) NOT NULL,
-  `last_passwd_gen` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
-  `stats_date_from` date DEFAULT NULL,
-  `stats_date_to` date DEFAULT NULL,
-  `stats_compare_from` date DEFAULT NULL,
-  `stats_compare_to` date DEFAULT NULL,
-  `stats_compare_option` int(1) unsigned NOT NULL DEFAULT 1,
-  `preselect_date_range` varchar(32) DEFAULT NULL,
-  `bo_color` varchar(32) DEFAULT NULL,
-  `bo_theme` varchar(32) DEFAULT NULL,
-  `bo_css` varchar(64) DEFAULT NULL,
-  `default_tab` int(10) unsigned NOT NULL DEFAULT '0',
-  `bo_width` int(10) unsigned NOT NULL DEFAULT '0',
-  `bo_menu` tinyint(1) NOT NULL DEFAULT '1',
-  `active` tinyint(1) unsigned NOT NULL DEFAULT '0',
-  `optin` tinyint(1) unsigned NOT NULL DEFAULT '1',
-  `id_last_order` int(10) unsigned NOT NULL DEFAULT '0',
-  `id_last_customer_message` int(10) unsigned NOT NULL DEFAULT '0',
-  `id_last_customer` int(10) unsigned NOT NULL DEFAULT '0',
-  `last_connection_date` date DEFAULT NULL,
-  `reset_password_token` varchar(40) DEFAULT NULL,
-  `reset_password_validity` datetime DEFAULT NULL,
-  PRIMARY KEY (`id_employee`),
-  KEY `employee_login` (`email`,`passwd`),
-  KEY `id_employee_passwd` (`id_employee`,`passwd`),
+  `id_employee` int(10) unsigned NOT NULL auto_increment, 
+  `id_profile` int(10) unsigned NOT NULL, 
+  `id_lang` int(10) unsigned NOT NULL DEFAULT '0', 
+  `lastname` varchar(255) NOT NULL, 
+  `firstname` varchar(255) NOT NULL, 
+  `email` varchar(255) NOT NULL, 
+  `passwd` varchar(2048) NOT NULL, 
```

Comment:
This column type does not fit the `Employee` class

Reply:
Fixed